### DISCUSSION TITLE
Please add 'detect next black scene' to the advanced toolbar

### DISCUSSION THREAD

**Discussion by sawoodpearce** [mifi/lossless-cut#1368] [Ideas]
Hi Mifi, please can a button be added to the advanced view which detects the **next black scene** from the cursor's current location?

I am often trimming video tapes where I don't need to detect cuts throughout which is very time-consuming, and just need the next blank scene from my current position to complete the edit.

Hopefully this can leverage most of the existing detection code and is just constraining it to start at the current time, and stop after one detection.

(Minor point, the tool is detecting 'blank' scenes rather than 'black'? As it seems to work great on blue frames from my VHS).

The tool is fantastic, keep up the great work. Thank you.

**Comment by sawoodpearce** [comment]
@mifi  would you consider this, perhaps? Just keen for your take on this at least. Thank you.

**Reply by mifi** [reply]
Hi. I think what you're looking for is this #1226 - see the discussion there. does that help?

**Reply by sawoodpearce** [reply]
Thanks for your reply. It looks similar, though appears to need more steps than finding the black scene myself with the jog/mouse wheel :) I'm just trying to trim a single segment typically, though I will give this a try. It is very helpful to know the detection tool already has a means to limit its scope.
